### PR TITLE
Scheduled event filtering

### PR DESCRIPTION
**Pull request by RSalaam**
WIP: Filtering data based upon selected filter choice. Currently not working properly.

### CODE REVIEW

File: components/events.jsx [hunk 1]
```diff
@@ -85,7 +182,7 @@ export default () => {
                       </button>
                     </td>
                   )}
-                {event.status === 'canceled' && <td>CANCELED</td>}
+                <td>{event.status}</td>
```
Comment (RSalaam) [inline components/events.jsx:185]: I like this change. I'm going to add a "status" column heading.

**Review by RSalaam — COMMENTED** [review]
This looks great! I'm going to see if I can fix the bug that prevents "load more" from disappearing when there's no more data to render. Also, the `past` and `future` filters aren't working correctly, BUT I think I know why. The `start_time` field value is not what the API expects. The `formatEventDateTime` function changes this value, so I'll fix that.

**Review by tristanoneil — APPROVED** [review]
👏 Looks great!

**Comment by RSalaam** [comment]
> 👏 Looks great!

Thank you!

**Pull request merged**